tests/tiglFuselageSegment.cpp

```diff
@@ -594,5 +635,47 @@ TEST_F(TiglFuselageSegment, GetSegmentVolume)
     ASSERT_GT(volume, 0.);
 }
 
+/***************************************************************************************************/
+
+TEST_F(TiglFuselageSegmentSimple, getSectionCenter)
+{
+    double eta = 0.0;
+
+    double pointX = 0;
+    double pointY = 0;
+    double pointZ = 0;
+
+    ASSERT_NE(TIGL_NULL_POINTER, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, &pointX, &pointY, &pointZ));
+    ASSERT_EQ(TIGL_SUCCESS, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, &pointX, &pointY, &pointZ));
+    EXPECT_NEAR(pointX, -0.5, 1e-15);
+    EXPECT_NEAR(pointY, 0, 1e-2);
+    EXPECT_NEAR(pointZ, 0, 1e-2);
+
+    eta = 0.5;
 
+    ASSERT_NE(TIGL_NULL_POINTER, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, &pointX, &pointY, &pointZ));
+    ASSERT_EQ(TIGL_SUCCESS, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, &pointX, &pointY, &pointZ));
+    EXPECT_NEAR(pointX, 0, 1e-15);
+    EXPECT_NEAR(pointY, 0, 1e-2);
+    EXPECT_NEAR(pointZ, 0, 1e-2);
+
+    eta = 1;
+
+    ASSERT_NE(TIGL_NULL_POINTER, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, &pointX, &pointY, &pointZ));
+    ASSERT_EQ(TIGL_SUCCESS, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, &pointX, &pointY, &pointZ));
+    EXPECT_NEAR(pointX, 0.5, 1e-15);
+    EXPECT_NEAR(pointY, 0, 1e-2);
+    EXPECT_NEAR(pointZ, 0, 1e-2);
+
+    ASSERT_EQ(TIGL_UID_ERROR, tiglFuselageGetSectionCenter(tiglHandle, "invalidUID", eta, &pointX, &pointY, &pointZ));
+
+    ASSERT_EQ(TIGL_NULL_POINTER, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, NULL, &pointY, &pointZ));
+    ASSERT_EQ(TIGL_NULL_POINTER, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, &pointX, NULL, &pointZ));
+    ASSERT_EQ(TIGL_NULL_POINTER, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, &pointX, &pointY, NULL));
+    ASSERT_EQ(TIGL_NULL_POINTER, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, NULL, NULL, &pointZ));
+    ASSERT_EQ(TIGL_NULL_POINTER, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, &pointX, NULL, NULL));
+    ASSERT_EQ(TIGL_NULL_POINTER, tiglFuselageGetSectionCenter(tiglHandle, "segmentD150_Fuselage_1Segment2ID", eta, NULL, NULL, NULL));
+
+    ASSERT_EQ(TIGL_NOT_FOUND, tiglFuselageGetSectionCenter(-1, "segmentD150_Fuselage_1Segment2ID", eta, &pointX, &pointY, &pointZ));
```

Comment:
Can you please add tests for the following use cases:
 - fuselageSegmentUID is NULL -> TIGL_NULL_POINTER
 - eta < 0 - >TIGL_MATH_ERROR
 - eta > 1 - >TIGL_MATH_ERROR